mktrayicon.c

```diff
@@ -334,27 +365,53 @@ gpointer watch_fifo(gpointer argv)
 				break;
 			}
 
-			// This block makes sure that the parameter after 'm' is ready to be processed
-			// We can't accept 2 straight commas, as it becomes ambiguous
+			/* This block makes sure that the parameter after 'm' is ready to be processed
+			 * We can't accept 2 straight commas, as it becomes ambiguous
+			 * We accept single quotes (') as another way to escape characters 
+			 * but they have to be even in number, or else it also becomes ambiguous
+			 */
 			int straight = 0;
 			int bars = 0;
+			int qEven = 1; // if single quotes are even -> we don't count anything 
```

Comment:
I would just call this `inQuoted`

Reply:
To not change the logic of the booleans, I will change it to outQuoted, which is the equivalent of qEven